user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Configuring_OpenSearch_Backups.md

```diff
@@ -22,18 +22,26 @@ Examples:
 
 - ...
 
-## Preparing and configuring the repositories
+## Setting up a shared folder
```

Comment:
I have renamed this section of the procedure as I felt the old title (Preparing and configuring the repositories) no longer fit the content. Feel free to change the title if you feel something else would be more fitting.